### DISCUSSION TITLE
Managing Variants on POS

### DISCUSSION THREAD

**Discussion by premudeshi** [alphabit-technology/erpnext-restaurant#24] [Q&A]
Hello there kind sir,

I was wondering how the POS system works with Variants. I have a dish that has over 3 different attributes, similar to a sandwich at Subway, and memorizing all the individual codes is not feasible. Is there a way to introduce a variant menu/variant form that the pos user can fill, and based on that, it would filter the product code?

Best,
Prem

**Comment by alphabit-technology** [comment]
Hello, let me analyze your question and give you an answer.
